### PR TITLE
Component/datatable enhancement1

### PR DESCRIPTION
**Pull request by itaipelles**
Just a little refactoring

### CODE REVIEW

**Comment by nirhart** [comment]
@lbelinsk can you please review it?

File: src/DataTable/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 |----------|----------|--------------|------------|-------------|
 | id | string | - | - | An id to pass to the table |
 | data | array | [] | - | The data to display |
-| columns | array | [] | - | Configuration of the table's columns. each column needs a title and a render method |
+| columns | array | [] | - | Configuration of the table's columns. each column needs a title and a render method. additional column options are width, minWidth and important. see example. |
```
Comment (itaibenda) [inline src/DataTable/README.md]: show the column object props in another table

File: src/DataTable/DataTable.js [hunk 2]
```diff
@@ -83,11 +83,21 @@ class DataTable extends React.Component {
       rowClasses.push(s.clickableDataRow);
     }
 
+    if (this.props.rowClass) {
+      rowClasses.push(this.props.rowClass);
```
Comment (itaibenda) [inline src/DataTable/DataTable.js]: the rowClass is already in the array, no need to add it twice

Comment (itaipelles) [inline src/DataTable/DataTable.js]: fixed in eaa49699bcdc90c51e158f40169745ac8a7f9a3f

**Comment by lbelinsk** [comment]
@nirhart i see Benda already viewed it...

**Pull request merged**